flipperdevices/flipperzero-firmware
# [FL-3012] Dolphin: new animation L2_Wake_up_128x64

# What's new

- Dolphin: new animation L2_Wake_up_128x64

# Verification 

- Compile and `flash_usb_full`
- Ensure that your dolphin level >= 2
- Check new animation

# Checklist (For Reviewer)
